pkgs/development/python-modules/black/default.nix

```
@@ -23,14 +23,19 @@ buildPythonPackage rec {
 
   # Don't know why these tests fails
   # Disable test_expression_diff, because it fails on darwin
+  # Disable Primer* tests because they depend on black beeing in PATH
   checkPhase = ''
     LC_ALL="en_US.UTF-8" pytest \
       --deselect tests/test_black.py::BlackTestCase::test_expression_diff \
       --deselect tests/test_black.py::BlackTestCase::test_cache_multiple_files \
-      --deselect tests/test_black.py::BlackTestCase::test_failed_formatting_does_not_get_cached
+      --deselect tests/test_black.py::BlackTestCase::test_failed_formatting_does_not_get_cached \
+      --deselect tests/test_black.py::PrimerCLITests::test_async_main \
+      --deselect tests/test_primer.py::PrimerLibTests::test_gen_check_output \
+      --deselect tests/test_primer.py::PrimerLibTests::test_process_queue \
+      --deselect tests/test_primer.py::PrimerCLITests::test_async_main
   '';
 
-  propagatedBuildInputs = [ attrs appdirs click toml aiohttp aiohttp-cors pathspec regex typed-ast ];
+  propagatedBuildInputs = [ attrs appdirs click toml aiohttp aiohttp-cors pathspec regex typed-ast mypy-extensions typing-extensions ];
```

These should be broken onto separate lines and sorted alphabetically.

```suggestion
  propagatedBuildInputs = [ 
    appdirs
    attrs 
    ...
  ];
```